sub-packages/bionemo-testing/src/bionemo/testing/harnesses/stop_and_go.py

```diff
@@ -234,10 +234,11 @@ def make_callbacks() -> Dict[Type[pl.Callback], pl.Callback]:
                 testing_callbacks.RaiseAfterMetadataCallback: testing_callbacks.RaiseAfterMetadataCallback(),
                 nl_callbacks.ModelCheckpoint: nl_callbacks.ModelCheckpoint(
                     save_last=True,
-                    monitor="reduced_train_loss",
+                    monitor="val_loss",
                     save_top_k=2,
                     every_n_train_steps=cls.val_check_interval,
                     always_save_context=True,
+                    filename="{epoch}-{step}-{val_loss:.2f}",
                 ),
```

Comment:
Even with these changes, I still get the `Last checkpoint is unfinished and cannot be used to resume the training` error

Reply:
This commit should now fix it.
https://github.com/NVIDIA/bionemo-framework/pull/302/commits/f2f6d60694c6443731b16f7356f07c89124a4ee1